mne/export/tests/test_export.py

```diff
@@ -46,6 +48,31 @@ def test_export_raw_eeglab(tmpdir):
     assert_allclose(raw.get_data(), raw_read.get_data())
 
 
+@pytest.mark.skipif(not _check_edflib_installed(strict=False),
+                    reason='edflib-python not installed')
+@pytest.mark.parametrize('format', ['edf', 'bdf'])
+def test_export_raw_edf_and_bdf(tmpdir, format):
+    """Test saving a Raw instance to EDF format."""
+    fname = (Path(__file__).parent.parent.parent /
+             "io" / "tests" / "data" / "test_raw.fif")
+    raw = read_raw_fif(fname)
+
+    # only test with EEG channels
+    raw.pick_types(eeg=True, eog=True, ecg=True, emg=True)
+
+    raw.load_data()
+    temp_fname = op.join(str(tmpdir), f'test.{format}')
+    raw.export(temp_fname)
+    raw.drop_channels([ch for ch in ['epoc']
+                       if ch in raw.ch_names])
+    raw_read = read_raw_edf(temp_fname, preload=True)
+    assert raw.ch_names == raw_read.ch_names
+    print(len(raw.times), len(raw_read.times))
+    assert_array_almost_equal(raw.times, raw_read.times, decimal=1)
```

Comment:
So I get this:

```
E    x: array([0.00e+00, 1.66e-03, 3.33e-03, ..., 2.40e+01, 2.40e+01, 2.40e+01])
E    y: array([0.00e+00, 1.67e-03, 3.33e-03, ..., 2.40e+01, 2.40e+01, 2.40e+01])
```

So I think its because of the machine division rounding up one of the numbers?

Reply:
maybe `assert_allclose(..., rtol=0, atol=1e-3)` would be better for checking times...  that should ensure everything is within 1 millisecond

Reply:
Done.

Reply:
Actually, there is still an error:

```
    assert_allclose(raw.times, raw_read.times, rtol=0, atol=1e-2)
E   AssertionError: 
E   Not equal to tolerance rtol=0, atol=0.01
E   
E   Mismatched elements: 8540 / 14400 (59.3%)
E   Max absolute difference: 0.02457271
E   Max relative difference: 0.00102393
E    x: array([0.000000e+00, 1.664960e-03, 3.329920e-03, ..., 2.397043e+01,
E          2.397210e+01, 2.397376e+01])
E    y: array([0.000000e+00, 1.666667e-03, 3.333333e-03, ..., 2.399500e+01,
E          2.399667e+01, 2.399833e+01])
```

These are the order of < 1 ms though at this point because of the machine division.

Reply:
Now fixed to 5 decimal places